/*
 * This file was automatically generated by EvoSuite
 * Fri Sep 06 14:01:57 GMT 2024
 */

package benchmarks.CLEVER.is_prime1.Eq;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import benchmarks.CLEVER.is_prime1.Eq.oldV;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class oldV_ESTest extends oldV_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      oldV oldV0 = new oldV();
      oldV0.NUMPRIMES = (-1);
      boolean boolean0 = oldV0.lib((-534), 3505);
      assertTrue(boolean0);
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      oldV oldV0 = new oldV();
      oldV0.primes = null;
      // Undeclared exception!
      try { 
        oldV0.lib(0, 1732);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("benchmarks.CLEVER.is_prime1.Eq.oldV", e);
      }
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      oldV oldV0 = new oldV();
      int[] intArray0 = new int[2];
      oldV0.primes = intArray0;
      intArray0[0] = (-1250);
      intArray0[1] = (-646);
      // Undeclared exception!
      try { 
        oldV0.lib((-1286), (-1286));
        fail("Expecting exception: ArrayIndexOutOfBoundsException");
      
      } catch(ArrayIndexOutOfBoundsException e) {
         //
         // 2
         //
         verifyException("benchmarks.CLEVER.is_prime1.Eq.oldV", e);
      }
  }

  @Test(timeout = 4000)
  public void test3()  throws Throwable  {
      oldV oldV0 = new oldV();
      int[] intArray0 = new int[8];
      oldV0.primes = intArray0;
      // Undeclared exception!
      try { 
        oldV0.lib((-3055), (-3055));
        fail("Expecting exception: ArithmeticException");
      
      } catch(ArithmeticException e) {
         //
         // / by zero
         //
         verifyException("benchmarks.CLEVER.is_prime1.Eq.oldV", e);
      }
  }

  @Test(timeout = 4000)
  public void test4()  throws Throwable  {
      oldV oldV0 = new oldV();
      boolean boolean0 = oldV0.lib(0, 0);
      assertFalse(boolean0);
  }

  @Test(timeout = 4000)
  public void test5()  throws Throwable  {
      oldV oldV0 = new oldV();
      boolean boolean0 = oldV0.lib(8, 8);
      assertFalse(boolean0);
  }

  @Test(timeout = 4000)
  public void test6()  throws Throwable  {
      oldV oldV0 = new oldV();
      boolean boolean0 = oldV0.lib(3697, 3697);
      assertTrue(boolean0);
  }

  @Test(timeout = 4000)
  public void test7()  throws Throwable  {
      oldV oldV0 = new oldV();
      boolean boolean0 = oldV0.client(3697);
      assertFalse(boolean0);
  }
}
